src/api/parser/src/utils/html/sanitize.js

```diff
@@ -50,6 +51,8 @@ module.exports = (dirty) => {
     allowedIframeHostnames: [
       'www.youtube.com',
       'player.vimeo.com',
+      'player.twitch.tv',
+      WEB_URL,
```

Comment:
I don't think this is where we want the `WEB_URL`, we need to strip put the `dev.to` from `parent='dev.to` and replace it with `parent={WEB_URL}`

Reply:
We need to make a `new URL()` out of the iframe URL in the parser, then break down its query params and modify the `parent` value, then get the `.href` to get the full URL again and put that in the iframe.